src/main/java/version1/gameUtil/settings/UIConfigurations.java

```diff
@@ -158,6 +158,7 @@ public static void configure(ModeScreen modeScreen){
                 .createMediumButton(mediumButtonText)
                 .createHardButton(hardButtonText)
                 .createGoToMenuButton(goToMenuText)
+                .createGoToEasyMazeButton(easyButtonText)
                 .createGoToMediumMazeButton(mediumButtonText)
```

Comment:
Same thing here. Lines 158 and 161. They are pretty much doing the same thing so one may need to leave